### PR TITLE
Create FAQ.md

### PR DESCRIPTION
**Pull request by whanamura**

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+>**Q:  Will it be kid friendly?**
+
+**A:**  Yes! Children under the age of 10 can come for free.  We will have a family camping area, where you can pitch your tent away from the music and late night gatherings. We will also have a family area with activities that young kids can enjoy--hands on creativity.
+
+Many of the activities will also appeal to all ages: from harvesting food for our meals, hiking up the local stream, going to the beach, to live music jams and yoga.
+
+However, you will be expected to be responsible for your children at all times. We won’t have babysitting and you can’t just drop your kids off in the family activities tent. So if you imagine yourself fully participating in workshops, parties, hack-a-thons and more, I think you should plan on bringing someone whose main focus is watching the kids.
+
+Q:  Will you have a tent big enough for 2 adults, 2 kids, and baby? Or hostel or RV (if able to rent RV)
```
Comment (benhylau) [inline FAQ.md]:
```suggestion
**Q:  Will you have a tent big enough for 2 adults, 2 kids, and baby? Or hostel or RV (if able to rent RV)**
```

File: FAQ.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+# FAQs 
+## (From emailed responses)
+
+
+>**Q:  Will it be kid friendly?**
```
Comment (benhylau) [inline FAQ.md]:
```suggestion
**Q:  Will it be kid friendly?**
```

File: FAQ.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+In an "un-conference" format, you could lead the kind of cross-sector talks that get to the heart of the matter.
+At the most time-intensive level, you could become a coordinator of some aspect of the camp--from running a creative track to booking lightning talks.
+Q:  What are some ways of furthering more "peer discovery" to initiate important and challenging conversations?  Maybe some of this has to do with the "ideological topology" of scheduling and layout?
+
+
+A:  This is our true goal.  To help participants connect internally with the values that drive their work; to connect with each other in new and surprising ways, and to connect with the environment.  So we'd love to have your input and help in creating the right “topology” or "containers" to enable that to happen.
+
+That's why we decided to camp--it breaks down barriers and encourages community and communication.  We'll be cooking, harvesting vegetables, washing dishes, hiking, hacking, dancing, making music, swimming, building sculptures—together.
+
+Q: I live in San Francisco and I don't own a car.  I would have to figure out transportation and lodging.
```
Comment (benhylau) [inline FAQ.md]:
```suggestion
**Q: I live in San Francisco and I don't own a car.  I would have to figure out transportation and lodging.**
```

File: FAQ.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+A:  We encourage radical sharing:  bring your talents, workshops, musical instruments, hacking projects--but let us know what you have in mind so we can help get you situated. 
+
+_NOTE: I would encourage you to register quickly once we open registration in early April.  We will be limiting attendance to the first 500 people._
+
+Q:  Can this event be attended by a mainly wheelchair-bound female techie ? I told her about the conference last year & she wanted to attend this year’s conference.  How inclusive will this event be?  Is this like a mini Burning Man w/ electric equipment/ generators hauled in?
+
+A:  I'm not sure. It's on a farm and ranch that is private land, so the terrain is variable.  It is not quite like a mini-Burning man--we will be near power and facilities--but living in tents in fields, eating in green houses, hiking to the beach.  We will check with the venue and try to make it as accessible as possible.
+
+Q  How can I be involved?
```
Comment (benhylau) [inline FAQ.md]:
```suggestion
**Q:  How can I be involved?**
```

File: FAQ.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+I think your best bet is to rent an RV in San Francisco (there is also an Air B-n-B for RVs..)  
+We will have spaces for RVs for a nominal RV parking charge.  Prices range from $100/night to $300+ for RVs that sleep 6. 
+
+Q:  I want to attend myself, plus encourage 5 of my coworkers/friends to join. We all work to decentralize the internet. I'm curious about the events, as we may want to have some events ourselves while there. 
+
+A:  We encourage radical sharing:  bring your talents, workshops, musical instruments, hacking projects--but let us know what you have in mind so we can help get you situated. 
+
+_NOTE: I would encourage you to register quickly once we open registration in early April.  We will be limiting attendance to the first 500 people._
+
+Q:  Can this event be attended by a mainly wheelchair-bound female techie ? I told her about the conference last year & she wanted to attend this year’s conference.  How inclusive will this event be?  Is this like a mini Burning Man w/ electric equipment/ generators hauled in?
```
Comment (benhylau) [inline FAQ.md]:
```suggestion
**Q:  Can this event be attended by a mainly wheelchair-bound female techie ? I told her about the conference last year & she wanted to attend this year’s conference.  How inclusive will this event be?  Is this like a mini Burning Man w/ electric equipment/ generators hauled in?**
```

File: FAQ.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+
+A:  I'm not sure. It's on a farm and ranch that is private land, so the terrain is variable.  It is not quite like a mini-Burning man--we will be near power and facilities--but living in tents in fields, eating in green houses, hiking to the beach.  We will check with the venue and try to make it as accessible as possible.
+
+Q  How can I be involved?
+You can come help us build, run, and strike the camp from July 15-22nd.  That could be anything from erecting the Domes to painting signs to decorating former mushroom growing rooms.  It will be a time for a smaller group to bond and build community.
+You could bring your talents and ideas and workshops.  This will be much more crowd-sourced content than the Summit.  
+In an "un-conference" format, you could lead the kind of cross-sector talks that get to the heart of the matter.
+At the most time-intensive level, you could become a coordinator of some aspect of the camp--from running a creative track to booking lightning talks.
+Q:  What are some ways of furthering more "peer discovery" to initiate important and challenging conversations?  Maybe some of this has to do with the "ideological topology" of scheduling and layout?
```
Comment (benhylau) [inline FAQ.md]:
```suggestion
**Q:  What are some ways of furthering more "peer discovery" to initiate important and challenging conversations?  Maybe some of this has to do with the "ideological topology" of scheduling and layout?**
```

File: FAQ.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+Q:  Will you have a tent big enough for 2 adults, 2 kids, and baby? Or hostel or RV (if able to rent RV)
+
+A:  Good question!  Our [Glamping tents](https://archive.org) come with one queen bed or two twin cots, so I think it would be a tight squeeze.  Each tent is $400 for the 3-4 nights.
+
+You could rent a room at the nearby Costanoa Lodge, which also has luxury glamping tents.
+The rooms at the nearby Pigeon Point Youth Hostel are already booked up, but you could check for cancellations.
+I think your best bet is to rent an RV in San Francisco (there is also an Air B-n-B for RVs..)  
+We will have spaces for RVs for a nominal RV parking charge.  Prices range from $100/night to $300+ for RVs that sleep 6. 
+
+Q:  I want to attend myself, plus encourage 5 of my coworkers/friends to join. We all work to decentralize the internet. I'm curious about the events, as we may want to have some events ourselves while there.
```
Comment (benhylau) [inline FAQ.md]:
```suggestion
**Q:  I want to attend myself, plus encourage 5 of my coworkers/friends to join. We all work to decentralize the internet. I'm curious about the events, as we may want to have some events ourselves while there. **
```

File: FAQ.md [hunk 8]
```diff
@@ -0,0 +1,62 @@
+# FAQs 
+## (From emailed responses)
```
Comment (benhylau) [inline FAQ.md]:
```suggestion
_From emailed responses_
```

File: FAQ.md [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+
+A:  Yes! Children under the age of 10 can come for free.  We will have a family camping area, where you can pitch your tent away from the music and late night gatherings. We will also have a family area with activities that young kids can enjoy--hands on creativity.
+
+Many of the activities will also appeal to all ages: from harvesting food for our meals, hiking up the local stream, going to the beach, to live music jams and yoga.
+
+However, you will be expected to be responsible for your children at all times. We won’t have babysitting and you can’t just drop your kids off in the family activities tent. So if you imagine yourself fully participating in workshops, parties, hack-a-thons and more, I think you should plan on bringing someone whose main focus is watching the kids.
+
+**Q:  Will you have a tent big enough for 2 adults, 2 kids, and baby? Or hostel or RV (if able to rent RV)**
+
+A:  Good question!  Our [Glamping tents](https://archive.org) come with one queen bed or two twin cots, so I think it would be a tight squeeze.  Each tent is $400 for the 3-4 nights.
```
Comment (benhylau) [inline FAQ.md]: @whanamura not sure if we want to list this option before confirming tent availability?

File: FAQ.md [hunk 10]
```diff
@@ -0,0 +1,62 @@
...
+
+You could rent a room at the nearby Costanoa Lodge, which also has luxury glamping tents.
+The rooms at the nearby Pigeon Point Youth Hostel are already booked up, but you could check for cancellations.
+I think your best bet is to rent an RV in San Francisco (there is also an Air B-n-B for RVs..)  
+We will have spaces for RVs for a nominal RV parking charge.  Prices range from $100/night to $300+ for RVs that sleep 6.
```
Comment (benhylau) [inline FAQ.md]: Unclear whether RV option is booked through DWeb Camp or independently arranged.

File: FAQ.md [hunk 11]
```diff
@@ -0,0 +1,62 @@
...
+I think your best bet is to rent an RV in San Francisco (there is also an Air B-n-B for RVs..)  
+We will have spaces for RVs for a nominal RV parking charge.  Prices range from $100/night to $300+ for RVs that sleep 6. 
+
+**Q:  I want to attend myself, plus encourage 5 of my coworkers/friends to join. We all work to decentralize the internet. I'm curious about the events, as we may want to have some events ourselves while there.**
+
+A:  We encourage radical sharing:  bring your talents, workshops, musical instruments, hacking projects--but let us know what you have in mind so we can help get you situated. 
+
+_NOTE: I would encourage you to register quickly once we open registration in early April.  We will be limiting attendance to the first 500 people._
+
+**Q:  Can this event be attended by a mainly wheelchair-bound female techie ? I told her about the conference last year & she wanted to attend this year’s conference.  How inclusive will this event be?  Is this like a mini Burning Man w/ electric equipment/ generators hauled in?**
```
Comment (benhylau) [inline FAQ.md:28]: Perhaps we can modify questions to make more generally applicable? Such as:

>Is DWeb Camp wheelchair accessible?

**Review by benhylau — CHANGES_REQUESTED** [review]
I added some commits to make formatting more consistent, then added comments on a few of the questions.

File: FAQ.md (hunk 9 above)
Comment (whanamura) [reply FAQ.md]: The MF has 146 of these tents, and their revenue model may depend on some of them getting rented.  In any case, I think we can stop calling them Glamping and describe them as tents that groups can rent, to stop making them seem exclusive.

File: FAQ.md (hunk 10 above)
Comment (whanamura) [reply FAQ.md]: I fixed that

File: FAQ.md (hunk 11 above)
Comment (whanamura) [reply FAQ.md:28]: I kind of like it that it is highly personal.

**Review by benhylau — APPROVED** [review]
This looks good to merge for me 👍

**Comment by benhylau** [comment]
@whanamura want to merge this?

**Pull request merged**